### ISSUE TITLE
bug with graphMin?

### ISSUE THREAD

**Issue by theBear** [FVANCOP/ChartNew.js#252]
I think there is an issue with the graphMin-function.

    graphMax : 100,
    graphMin : 0,
    yAxisMinimumInterval : 5

...will lead to this correct version:

![correct](https://cloud.githubusercontent.com/assets/7254827/6430693/45f79cc4-c015-11e4-98cd-71e7c9259263.png)

...but if I change graphMin to 1:

    graphMax : 100,
    graphMin : 0,
    yAxisMinimumInterval : 5

this appears:

![wrong](https://cloud.githubusercontent.com/assets/7254827/6430694/58fb717e-c015-11e4-8e29-503ef2655b67.png)

As you can see, the "0" ist still there, but my values begin at 1. Or this there another option to fix this?

**Comment by FVANCOP** [comment]
You set graphMax to 100, graphMin to 1 and yAxisMinimumInterval to 5 

-> You have to display at least graphMax and graphMin;
-> The interval between 2 values must be 5 

If the chart starts at 1 (instead of 0), with an interval of 5, it will display 1, 6, 11, ..... 96 and 101 which is not what the users want. So I don't think that is is bad to start at 0 even if graphMin is initialised to 1. If you don't want to see the 0, you should set graphMin to 5 or initialise yAxisMinimumInterval to 1.

**Comment by theBear** [comment]
I agree, that the most users don't want a sequence like "1, 11, 21", but let's take a look two steps further, if you like to focus the chart to a specified "area" EVEN, if the highest and lowest peak wouldn't be shown. I know in statistics this situation is sometimes quite common. 

Example values: 2, 54248, 54230, 54100, 200000

You'd like to show a fine scale of "54248, 54230, 54100", but not the outliers "2" and "200000". If you put all of these 5 values into a simple chart, "54248, 54230, 54100" look like a straight line. This focusing ist exactly the same circumstance I have above.

Also: "you should set graphMin to 5 or initialise yAxisMinimumInterval to 1" is just an auxiliary release, not the solution of this issue.

**Comment by FVANCOP** [comment]
It is exact that, in statistics, you sometimes want to exclude outliers/extreme values. But, if you want to exclude some values, you need to specify an algorithm/function that will point out those outliers/extreme values. So I would say, this is the responsability of the programmer to exclude those values from the data before the graph function is called.

**Comment by theBear** [comment]
It will work with the example I gave. There you could trim the first and the last value. But it won't work with a sequence like: 2, 54248, 54230, 4, 7, 54100, 200000, 54250 and e.g. a focus between 54100 and 54300.
